t/lib/CustomCensor.pm

```diff
@@ -0,0 +1,11 @@
+package CustomCensor;
```

Comment:
It appears as if you created this and didn't use it. Am I missing something?

That's my only question. Otherwise, this is 👍 from me. Really like the changes you made!

Reply:
ooops. Looks like I started thinking I'd use it and code/thoughts went elsewhere. Removed!